test/alternator/test_returnconsumedcapacity.py

```diff
@@ -190,3 +190,30 @@ def test_number_magnitude_key(test_table_sn):
             response = test_table_sn.put_item(Item={'p': p, 'c': num, 'a': x, 'val2': val2 + 'a'*10}, ReturnConsumedCapacity='TOTAL')
             assert 2 == response.get('ConsumedCapacity')["CapacityUnits"]
 
+# the simple delet item validate that when deleting a short item
+# from a table we will get 1 WCU
+def test_simple_delete_item(test_table_sb):
+    p = random_string()
+    val = random_string()
+    c = random_bytes()
+    test_table_sb.put_item(Item={'p': p, 'c': c, 'att': val}, ReturnConsumedCapacity='TOTAL')
+    response = test_table_sb.delete_item(Key={'p': p, 'c': c}, ReturnConsumedCapacity='TOTAL')
+    assert 'ConsumedCapacity' in response
+    assert 1 == response['ConsumedCapacity']["CapacityUnits"]
```

Comment:
Please also a test function for deleting a non-existent item (i.e., random key where you didn't PutItem first) - it should still cost one unit (I think).

Reply:
will do